src/scripts/main.js

```diff
@@ -1,3 +1,33 @@
 'use strict';
 
-// write code here
+[...document.querySelectorAll('li')].forEach((li) => {
+  const span = document.createElement('span');
+
+  span.prepend(li.firstChild);
+  li.prepend(span);
+});
+
+[...document.querySelectorAll('ul')].forEach(el => {
+  el.addEventListener('click', (event) => {
+    event.stopPropagation();
+
+    const targetNode = event.target;
+
+    if (targetNode.tagName.toLowerCase() === 'span'
+    && targetNode.parentNode.querySelector('ul')) {
+      const ul = targetNode.parentNode.childNodes[1];
+
+      ul.style.transformOrigin = 'top';
+      ul.style.transition = '0.5s';
+      ul.style.overflow = 'hidden';
```

Comment:
hiding element with overflow = 'hidden'; and maxHeight = '0' looks like a hack, maybe it's better to use hidden attribute?